frontend/spec/controllers/spree/home_controller_spec.rb

```diff
@@ -16,11 +16,12 @@
     end
 
     context "different layout specified in config" do
-      before { Spree::Config.layout = 'layouts/application' }
+      before { Spree::Config.layout = 'layouts/my_super_awesome_layout' }
 
       it "renders specified layout" do
-        get :index
-        expect(response).to render_template(layout: 'layouts/application')
+        expect {
+          get :index
+        }.to raise_error(ActionView::MissingTemplate, %r{Missing template layouts/my_super_awesome_layout})
```

Comment:
I think we should not change the spec like this. Is there a way to tell the dummy app the view paths like you did with the assets?

Reply:
I do, but can send a separate PR